rubix-bookkeeper/src/test/java/com/qubole/rubix/bookkeeper/validation/TestFileValidatorVisitor.java

```diff
@@ -122,6 +149,106 @@ public void testValidatorFileVisitor_someMdExists_multiDepth() throws IOExceptio
     runAndVerifyValidator(conf, maxDepth, maxDirs, maxFiles, mdSkip);
   }
 
+  @Test
+  public void testValidatorFileVisitor_allValidCachedFiles() throws TException, IOException
```

Comment:
There's a lot of repeated code in these tests for creating the file + getting the status + setting the blocks as cached. Would it be possible to refactor these steps into another method?

Reply:
Done

Reply:
LGTM